### ISSUE TITLE
Long clause template names intersect with template logo

### ISSUE THREAD

**Issue by Michael-Grover** [accordproject/web-components#184]
# Bug Report 🐛

Long clause template names intersect with the template logo, like this:
![image](https://user-images.githubusercontent.com/40569793/92003322-e37a7580-ed0e-11ea-840b-b7cb366964df.png)

## Expected Behavior
Template names should wrap to a second line if they can not fit on the first line without overlapping the logo

**Comment by ishitajaiswal4m** [comment]
@irmerk can i work on this issue ?

**Comment by naveen8801** [comment]
@irmerk Hii ,  i  wanna work on it , is this issue still open ?

**Comment by Michael-Grover** [comment]
@naveen8801 it looks like there has been no activity on this issue in the last three months. Thanks for volunteering! I will assign it to you.

**Comment by naveen8801** [comment]
@Michael-Grover  , thanks

**Comment by naveen8801** [comment]
Hii @Michael-Grover  , i have made some pull requests but some tests are it failed in some tests . Need help !

**Comment by Michael-Grover** [comment]
@DianaLease, @jeromesimeon, or @irmerk may be able to help

**Comment by DianaLease** [comment]
Closed by https://github.com/accordproject/web-components/pull/239
